contrib/generate-dbus-interface.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/python3
+# pylint: disable=invalid-name,missing-docstring
+#
+# Copyright (C) 2022 Richard Hughes <richard@hughsie.com>
+#
+# SPDX-License-Identifier: LGPL-2.1+
+
+import argparse
+import xml.etree.ElementTree as ET
+
+
+def _remove_docs(parent):
+
+    namespaces = {"doc": "http://www.freedesktop.org/dbus/1.0/doc.dtd"}
+    for node in parent.findall("doc:doc", namespaces):
+        parent.remove(node)
+    parent.text = ""
+
+
+if __name__ == "__main__":
+
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        "-r",
+        "--releases",
+        type=int,
+        default=5,
+    )
+    parser.add_argument(
+        "filename_src", action="store", type=str, help="metainfo source"
```

Comment:
just as a personal preference, why not make this `src` and `dst`?  It seems needlessly long.

Reply:
Agreed, done.